app/src/main/java/mozilla/lockbox/presenter/AuthPresenter.kt

```diff
@@ -39,7 +40,9 @@ class AuthPresenter(
 ) : Presenter() {
 
     override fun onViewReady() {
+        // taking values, or this ends up firing multiple times!
```

Comment:
Nit: remove comment

Reply:
I can remove the comment, though I think it has value to indicate a dragon to future us.

Reply:
that's fine with me 😎